_config.yml

```diff
@@ -1,2 +1,23 @@
----
-remote_theme: randomblock1/minimal
+- name: Setup Node.js environment
```

Comment:
This isn't a GitHub Actions file. It's a Jekyll configuration. This would just break the website.

Also, we don't need Node.js, so I'm not sure why you added this anyway.